python/ql/src/external/DuplicateBlock.ql

```diff
@@ -16,19 +16,7 @@
  */
 
 import python
-import CodeDuplication
 
-predicate sorted_by_location(DuplicateBlock x, DuplicateBlock y) {
-  if x.sourceFile() = y.sourceFile()
-  then x.sourceStartLine() < y.sourceStartLine()
-  else x.sourceFile().getAbsolutePath() < y.sourceFile().getAbsolutePath()
-}
-
-from DuplicateBlock d, DuplicateBlock other
-where
-  d.sourceLines() > 10 and
-  other.getEquivalenceClass() = d.getEquivalenceClass() and
-  sorted_by_location(other, d)
-select d,
-  "Duplicate code: " + d.sourceLines() + " lines are duplicated at " +
-    other.sourceFile().getShortName() + ":" + other.sourceStartLine().toString()
+from BasicBlock d
+where none()
```

Comment:
I'm not sure why we've half-removed the query. What is the purpose of removing CodeDuplication.qll, but keeping all the queries that depend on it? I think we should either remove these queries completely, or keep them working.

Reply:
Is there a technical reason to remove CodeDuplication.qll at this time?

Reply:
This PR is simply following what Java did, which is based on internal discussion on slack. The main idea is to not break any query-suites that still reference the code duplication queries.

Reply:
Ah I see, if the data isn't in the database then it's safe to delete this file.